spec/std/exception/call_stack_spec.cr

```diff
@@ -13,16 +13,21 @@ describe "Backtrace" do
     _, output, _ = compile_and_run_file(source_file)
 
     # resolved file line:column
-    output.should match(/^#{source_file}:3:10 in 'callee1'/m)
-    output.should match(/^#{source_file}:13:5 in 'callee3'/m)
+    {% if flag?(:win32) %}
+      output.should match(/^#{Regex.escape(source_file)}:3 in 'callee1'/m)
```

Comment:
Maybe add a brief comment that column numbers are not expected to work on windows?